e2e/cypress/integration/journals.test.js

```diff
@@ -1,5 +1,28 @@
 import { onlyOn } from '@cypress/skip-test';
 
+describe('Journal Detail', () => {
+  onlyOn('headless', () => {
+    it('matches image snapshot', () => {
+      cy.registerRoute();
+      cy.visit('/journals/1213103');
+      cy.waitForRoute();
+      cy.matchSnapshots('JournalDetail');
+    });
+  });
+});
+
+describe('Journal Search', () => {
+  onlyOn('headless', () => {
+    it('matches image snapshot', () => {
+      cy.registerRoute();
+      cy.visit('/journals');
+      cy.waitForRoute();
+      cy.waitForSearchResults();
+      cy.matchSnapshots('JournalsSearch');
+    });
+  });
+});
+
```

Comment:
your snapshots are missing `papers_count`

Reply:
This will be added in second step of completing the task ([explained here](https://github.com/cern-sis/issues-inspire/issues/5#issuecomment-1160234416))